src/components/StarRating.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+function StarRating(props){
+    let stars = props.stars;
+    let starRating = []
+    for(var i =0; i < 5; i++){
```

Comment:
This is pretty good. You could also use 2 loops, one for the stars and one for the empty stars